third_party/googleapis/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
can we checkout this dir during the build.
reason - keep everything under version control

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/1/commits/21e3912b2060d96237a39395bbf668e06563ca34